app/models/user.rb

```diff
@@ -28,4 +28,10 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
   has_many :articles, dependent: :destroy
   has_many :comments, dependent: :destroy
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "X50>" }, default_url: "/images/:style/missing.png"
```

Comment:
> - resize only if target image is larger than the specified dimensions
< - resize only if target image is smaller than the specified dimensions
! - resize to specified dimensions (may lose aspect ratio)
% - dimensions specified are considered as percentage of original dimensions
# - used to create nice square thumbnails by cropping the original image